app/views/rewards/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Reward page</h1>
```

Comment:
Great question! I was thinking of rewards as like a full page spread. 

So a user is browsing looking at rewards... and then they click to check one out and get a sense of what their reward would look like. Kind of like Amazon. You are browsing books. And then you click one to see the details.

Do you agree or disagree?